recipes/the-silver-searcher/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "the-silver-searcher" %}
```

Comment:
Seems most package managers use `_`. Would do the same here too.

ref: https://github.com/ggreer/the_silver_searcher#installing